src/NuGet.Core/NuGet.Packaging/PackagingConstants.cs

```diff
@@ -44,6 +44,18 @@ public static class Folders
                 Analyzers,
                 Source
             };
+
+            public static string[] SupportFrameworks { get; } = new string[] {
```

Comment:
Can we use `ImmutableArray` or `IReadOnlyCollection` or `IReadOnlyList` instead of `string[]`?  Right now, someone can do `SupportFrameworks[0] = "whatever" and suddenly NuGet is going to behave differently.